### PR TITLE
[ADD][13.0] add partner_address_two_lines module

### PR DESCRIPTION
**Pull request by Tonow-c2c**
Before:
image

----

After :

image

### CODE REVIEW

**Review by vrenaville — APPROVED** [review]

File: partner_address_two_lines/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This module change the reports layout
```
Comment (jcoux) [inline partner_address_two_lines/readme/DESCRIPTION.rst]:
```suggestion
This module changes the reports layout.
```

**Review by jcoux — APPROVED** [review]
Little suggestion, but LGTM :+1:

**Review by vrenaville — APPROVED** [review]

**Review by newtratip — APPROVED** [review]
LGTM :+1:

File: partner_address_two_lines/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+from odoo import models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    def _get_contact_name(self, partner, name):
+        if self.env.context.get("_two_lines_partner_address"):
+            return "{}\n {}".format(
```
Comment (NL66278) [inline partner_address_two_lines/models/res_partner.py:12]: What if the partner does neither have a commercial_company_name nor a parent?

File: partner_address_two_lines/models/res_partner.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    def _get_contact_name(self, partner, name):
+        if self.env.context.get("_two_lines_partner_address"):
```
Comment (NL66278) [inline partner_address_two_lines/models/res_partner.py:11]: If you return the result from super when _two_lines_partner_address is NOT in context, you do not need the else and save an indentation level for the remainder of the method.

**Comment by NL66278** [comment]
@Tonow-c2c I added some comments and a question last year. Will you be taking those up? Would be nice to merge this.

**Pull request merged**